docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -18,7 +18,9 @@ Once you understand the concepts and examples on this page, read about :ref:`Ans
 Creating valid variable names
 =============================
 
-Not all strings are valid Ansible variable names. A variable name must start with a letter, and can only include letters, numbers, and underscores. `Python keywords`_ and :ref:`playbook keywords<playbook_keywords>` are not valid variable names.
+Not all strings are valid Ansible variable names. A variable name can only include letters, numbers, and underscores. `Python keywords`_ and :ref:`playbook keywords<playbook_keywords>` and :ref:`playbook keywords<playbook_keywords>` are not valid variable names. A variable name cannot begin with a number.
```

Comment:
I think the last `and :ref:playbook keywords<playbook_keywords>` is a duplicate.

Reply:
Oh, good catch, thanks @richm.